app/src/main/java/org/mozilla/fenix/FeatureFlags.kt

```diff
@@ -89,6 +89,11 @@ object FeatureFlags {
      */
     val historyImprovementFeatures = Config.channel.isNightlyOrDebug
 
+    /**
+     * Separates history into local and synced from other sources.
+     */
+    val syncedHistoryFeature = Config.channel.isNightlyOrDebug
```

Comment:
```suggestion
    val syncedHistoryFeature = Config.channel.isDebug
```

Probably too early to enable for Nightly.

Reply:
fixed